scripts/npm-unit-tests.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
+# This script runs npm unit tests for the front end
+
+FRONT_END_DIR='src/frontend'
```

Comment:
In future, it might be nice if the dir is passed into the script as an argument, so we can make the script re-usable across repos. But this is fine for now / will be swept up as the pipeline refactor continues.

Reply:
ok yes noted.